src/requirements/data/minors/ineq.ts

```diff
@@ -0,0 +1,231 @@
+import { CollegeOrMajorRequirement } from '../../types';
+import { includesWithSubRequirements } from '../checkers-common';
+
+const inequalityRequirements: readonly CollegeOrMajorRequirement[] = [
+  {
+    name: 'Inequality Core Course',
+    description:
+      'This course reviews contemporary approaches to understanding social' +
+      ' inequality and the processes by which it comes to be seen as legitimate,' +
+      ' natural, or desirable. Offered every spring semester.',
+    source: 'https://inequality.cornell.edu/minor-overview-and-requirements',
+    checker: includesWithSubRequirements(['PAM 2208', 'SOC 2208', 'DSOC 2090']),
+    fulfilledBy: 'courses',
+    perSlotMinCount: [1],
+    slotNames: ['Course'],
+  },
+  {
+    name: 'Inequality Overview Course',
+    description: 'An Overview Course',
+    source: 'https://inequality.cornell.edu/minor-overview-and-requirements',
+    checker: includesWithSubRequirements([
+      'SOC 2220',
+      'DSOC 2220',
+      'PAM 2220',
+      'ILROB 2220',
+      'PHIL 1950',
+      'AMST 2225',
+      'GOVT 2225',
+      'DSOC 3700',
+      'SOC 3710',
+      'PAM 3370',
+      'SOC 3370',
+      'ILROB 6260',
+      'PAM 4470',
+      'SOC 4470',
+      'ILRLE 4426',
+      'ECON 3450',
+    ]),
+    fulfilledBy: 'courses',
+    perSlotMinCount: [1],
+    slotNames: ['Course'],
+  },
+  {
+    name: 'Inequality Electivess',
+    description:
+      'Four electives chosen from the full list of elective courses.' +
+      'If enrolled in the Health Equity Track, two of those four electives must ' +
+      'be electives labelled with the indicated health-equity focus. The other two' +
+      'can be chosen from the broader list and can include other health-equity courses.',
+    source: 'https://inequality.cornell.edu/minor-overview-and-requirements',
```

Comment:
I think https://inequality.cornell.edu/spring-2022-minor-course-list might be a better source because this is exactly where we got our data from. However, the current source does give more context and will probably stay current. Completely up to you what you want to do about this.

Reply:
sike don't do this